modules/health_quest/app/services/health_quest/patient_generated_data/questionnaire_response/factory.rb

```diff
@@ -63,7 +63,7 @@ def search(filters)
         # @param data [Hash] questionnaire answers and appointment data hash.
         # @return [FHIR::DSTU2::Patient::ClientReply] an instance of ClientReply
         #
-        def create_questionnaire_response(data)
+        def create(data)
```

Comment:
why shorten the name?

Reply:
Just following the already established convention. We already have methods named `get` and `search` inside the class so `create_<this_thing>` seemed redundant.